### PR TITLE
Remove node build from semantics

### PR DESCRIPTION
**Pull request by ehildenb**
The Node build is fully subsumed by the Web3 build now, so no need to keep it around, which should allow for some simplifications in the blockchain-k-plugin as well.

### CODE REVIEW

File: evm.md [hunk 1]
```diff
@@ -461,63 +460,6 @@ The `CallOp` opcodes all interperet their second argument as an address.
...
 
-### Helpers
-
--   `#addr` decides if the given argument should be interpreted as an address (given the opcode).
-
-```k
-    syntax InternalOp ::= "#load" "[" OpCode "]"
- // --------------------------------------------
-    rule <k> #load [ OP:OpCode ] => #loadAccount #addr(W0) ... </k>
-         <wordStack> (W0 => #addr(W0)) : WS </wordStack>
```
Comment (dwightguth) [inline evm.md:472]: Removing these three rules entirely looks suspicious.

**Comment by ehildenb** [comment]
Should be ready for re-review, I've tested locally, and the only thing I'm not convinced of is that the proofs will all pass.

File: evm.md [hunk 2]
```diff
@@ -461,61 +460,27 @@ The `CallOp` opcodes all interperet their second argument as an address.
...
-    rule #addr?(SELFDESTRUCT) => true
-    rule #addr?(EXTCODEHASH)  => true
-    rule #addr?(OP)           => false requires (OP =/=K BALANCE) andBool (OP =/=K SELFDESTRUCT) andBool (OP =/=K EXTCODEHASH)
-
-    syntax Bool ::= "#code?" "(" OpCode ")" [function]
- // --------------------------------------------------
-    rule #code?(EXTCODESIZE)  => true
-    rule #code?(EXTCODECOPY)  => true
-    rule #code?(OP)           => false requires (OP =/=K EXTCODESIZE) andBool (OP =/=K EXTCODECOPY)
+    rule <k> #addr [ OP:OpCode ] => . ... </k> [owise]
```
Comment (dwightguth) [inline evm.md:483]: pretty sure the java backend doesn't understand owise on non-function rules.

Comment (ehildenb) [reply evm.md]: @dwightguth I converted to an explicit side condition.

**Comment by dwightguth** [comment]
Looks fine. I wonder if we can make that side condition faster by converting it to structural matching, but we can worry about that later.

**Pull request merged**